src/graph/validator/MutateValidator.cpp

```diff
@@ -233,6 +238,32 @@ Status InsertEdgesValidator::prepareEdges() {
     auto valsRet = SchemaUtil::toValueVec(row->values());
     NG_RETURN_IF_ERROR(valsRet);
     auto props = std::move(valsRet).value();
+
+    std::vector<Value> entirePropValues;
+    for (size_t j = 0; j < fieldNum; ++j) {
+      auto *field = schema_->field(j);
+      auto propName = entirePropNames_[j];
```

Comment:
It seems that the `entirePropNames_` is not necessary, why not use `schema_->field(j)->name()` here directly. And you needs not to save it as the field of `MutateValidator`

Reply:
ACK.

Reply:
Also used in `InsertEdgesValidator::toPlan()`.